spec/operators/skipLast-spec.ts

```diff
@@ -0,0 +1,155 @@
+import {expect} from 'chai';
+import * as Rx from '../../dist/cjs/Rx';
+declare const {hot, cold, asDiagram, expectObservable, expectSubscriptions};
+
+const Observable = Rx.Observable;
+
+/** @test {takeLast} */
+describe('Observable.prototype.skipLast', () => {
+  asDiagram('skipLast(2)')('should skip two values of an observable with many values', () => {
+    const e1 =  cold('--a-----b----c---d--|');
+    const e1subs =   '^                   !';
+    const expected = '-------------a---b--|';
```

Comment:
There is something really wrong about this marble diagram. The bottom `a` should *not* be emitted at the same time as the top `c`, because when `c` occurs, we don't actually yet know that it's the second-last emission. We can only know that when the `e1` completes. So the expected should be `(ab|)` when the source completes.